connectors/crowdstrike/connector/data_handler.py

```diff
@@ -6,13 +6,15 @@
 
 def timestamp_conv(time_string):
     """ Convert date time to epoch time format """
-    if 'T' in time_string:
-        time_pattern = "%Y-%m-%dT%H:%M:%S"
-    else:
-        time_pattern = "%Y-%m-%d %H:%M:%S"
-    epoch = datetime.datetime(1970, 1, 1)
-    converted_time = int(((datetime.datetime.strptime(str(time_string)[:19],
-                                                      time_pattern) - epoch).total_seconds()) * 1000)
+    converted_time = None
+    if time_string:
```

Comment:
Was it possible that `time_string` came in as an empty string? Just wondering why this condition is needed.

Reply:
Sometime the datasource  doesn’t have published_date fro vlun. In that case when we calll for timestamp_conv, it throws an error.